ammo-be/src/routers/bulletRouter/tests/index.ts

```diff
@@ -0,0 +1,26 @@
+import express from 'express';
+import { ioServer } from 'index';
+import bulletSchema from 'validators/bulletValidator';
+import { Bullet } from 'shared/typings/Bullet';
+
+const bulletRouter = express.Router();
+
+bulletRouter.post(
+    '/',
+    (req, res): express.Response => {
+        try {
+            const potentialBullet: Bullet = { ...req.body.data };
```

Comment:
Why not use 'bullet'? Could remove a few characters

Reply:
Be cause at this specific point the bullet is still not validated, hence why it's a _potential_ one. It's only to point the difference with the actual bullet after join's has validated.